### DISCUSSION TITLE
大佬们 新增的 “Aria2 RPC“如何使用

### DISCUSSION THREAD

**Discussion by lpxks** [xifangczy/cat-catch#406] [Q&A]
文档看的不是很明白

**Comment by xifangczy** [comment]
我推荐 https://github.com/agalwood/Motrix 项目

在设置 - 进阶设置 可以填写 RPC端口和密钥

![rpc](https://github.com/user-attachments/assets/d9e41243-3093-4bcd-813c-2ab8ca8a2952)

注意 Motrix 默认rpc端口是 16800 你需要在猫抓设置 地址改为 `http://localhost:16800/jsonrpc` 然后打开`启用 Aria2 RPC`
在资源列表会有 ![aria2](https://github.com/user-attachments/assets/c997845b-cfe2-4855-98b3-caf7ea8ed8ff) 图标
在打开Motrix的状态 点击图标 会把下载任务提交给 Motrix 下载。

**Reply by lpxks** [reply]
谢谢，但是好像下载不了m3u8格式的视频 可能适合下载磁力链接的吧
